src/material-experimental/mdc-dialog/_dialog-theme.scss

```diff
@@ -0,0 +1,14 @@
+@import '../mdc-helpers/mdc-helpers';
+@import '@material/dialog/mixins.import';
+
+@mixin mat-mdc-dialog-theme($theme) {
+  @include mat-using-mdc-theme($theme) {
+    @include mdc-dialog-core-styles($mat-theme-styles-query);
```

Comment:
The pattern we've been using in other places is to pass in the query as a named parameter (e.g. `@include core-styles($query: $theme-query)`) in order to avoid errors if they add a new parameter.